app/vmalert/config/config.go

```diff
@@ -208,24 +231,24 @@ type ValidateTplFn func(annotations map[string]string) error
 var cLogger = &log.Logger{}
 
 // ParseSilent parses rule configs from given file patterns without emitting logs
-func ParseSilent(pathPatterns []string, validateTplFn ValidateTplFn, validateExpressions bool) ([]Group, error) {
+func ParseSilent(pathPatterns []string, validateTplFn ValidateTplFn, validateExpressions bool, defaultGroupInterval time.Duration) ([]Group, error) {
 	cLogger.Suppress(true)
 	defer cLogger.Suppress(false)
 
 	files, err := readFromFS(pathPatterns)
 	if err != nil {
 		return nil, fmt.Errorf("failed to read from the config: %s", err)
 	}
-	return parse(files, validateTplFn, validateExpressions)
+	return parse(files, validateTplFn, validateExpressions, defaultGroupInterval)
 }
 
 // Parse parses rule configs from given file patterns
-func Parse(pathPatterns []string, validateTplFn ValidateTplFn, validateExpressions bool) ([]Group, error) {
+func Parse(pathPatterns []string, validateTplFn ValidateTplFn, validateExpressions bool, defaultGroupInterval time.Duration) ([]Group, error) {
```

Comment:
Would be nice to move the flag with default eval interval to config. But we should have done this from the very beginning. Now, evalInteval kinda belongs logically to main, as main decides when to evaluate something. So I think we shouldn't propagate `defaultGroupInterval` param to config. Let main.go to decide the default.

Reply:
Okay, brought `defaultGroupInterval` back to main.